.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # BotKube maintainers (email: dev-team@botkube.io)
 # Maintainers are listed alphabetically
 
-* @ezodude @mszostok @pkosiec @PrasadG193
+* @madebyrogal @ezodude @mszostok @pkosiec @PrasadG193
```

Comment:
**nit** as stated above "# Maintainers are listed alphabetically", so please update the list :) 

```suggestion
* @ezodude @madebyrogal @mszostok @pkosiec @PrasadG193
```

Reply:
Done. Thanks